_sources/vectors_in_cpp/mixed_up_code.rst

```diff
@@ -0,0 +1,292 @@
+    =====
+                    words[i][j] = 'A';
+    =====
+                }
+    =====
+            }
+    =====
+        }
+    =====
+    }
+
+.. parsonsprob:: vectors_in_cpp_mixed_up_code_3
+    :numbered: left
+    :adaptive:
+
+    Suppose ``album`` has already been defined as ``vector<string> album = {"imagine", "needy", "NASA", "bloodline", "fake smile", "bad idea", "make up", "ghostin", "in my head", "7 rings", "thank u, next", "break up with your girlfriend, i'm bored"}``
+    Construct a block of code that counts how many songs in ``album`` start with b. Put the necessary blocks of code in the correct order.
+    -----
+    #include <vector>
+    #include <string>
+    using namespace std;
+    =====
+    int main() {
+    =====
+        vector<string> album = {"imagine", "needy", "NASA", "bloodline", "fake smile", "bad idea", "make up", "ghostin", "in my head", "7 rings", "thank u, next", "break up with your girlfriend, i'm bored"};
+    =====
+        int count = 0;
+    =====
+        for (int i = 0; i < album.size(); i++) {
+    =====
+            if (album[i][0] == 'b') {
+    =====
+            if (album[i] == 'b') { #paired
+    =====
+                count++;
+    =====
+            }
+    =====
+        }
+    =====
+    }
+
+.. parsonsprob:: vectors_in_cpp_mixed_up_code_4
+    :numbered: left
+    :adaptive:
+
+    Suppose you have the following two vectors to describe the weekly forecast ``vector<double> temps = {82.0, 76.8, 74.3, 58.8, 79.2, 73.4, 80.1}``,
+    ``vector<double> precip = {0.00, 0.30, 0.60, 0.90, 0.10, 0.20, 0.80}``. Your family will go to the beach if the temperature at least 75 degrees and the chance
+    of precipitation is less than 50%.  Construct a block of code that counts how many days your family can hit the beach on your vacation.
+    Put the necessary blocks of code in the correct order.
+    -----
+    #include <vector>
+    using namespace std;
+    =====
+    int main() {
+    =====
+        vector<double> temps = {82.0, 76.8, 74.3, 58.8, 79.2, 73.4, 80.1};
+    =====
+        vector<double> precip = {0.00, 0.30, 0.60, 0.90, 0.10, 0.20, 0.80};
+    =====
+        int count = 0;
+    =====
+        for (int i = 0; i < 7; i++) {
+    =====
+            if (temps[i] >= 75.0 && precip[i] < 0.50) {
+    =====
+            if (temps[i] > 75.0 || precip[i] < 0.50) { #paired
+    =====
+                count++;
+    =====
+            }
+    =====
+        }
+    =====
+    }
+
+.. parsonsprob:: vectors_in_cpp_mixed_up_code_5
+    :numbered: left
+    :adaptive:
+
+    Suppose you have the following vector ``vector<string> nouns = {"cereal", "Cocoa Puffs", "Mario", "luigi", "Aerosmith"}``.
+    Construct a block of code that creates a vector of the proper nouns in ``nouns`` called ``proper``.
+    Use the ``isupper`` function to check if a letter is uppercase. Put the necessary blocks of code in the correct order.
+    -----
+    #include <vector>
+    #include <string>
+    using namespace std;
```

Comment:
Can't see <vector> and <string>

Reply:
All of the "<" and ">" should now show, the formatting on my end was made incorrectly, sorry! I know that all of the questions on this parsons page had a similar problem.

Reply:
It looks good now, thank you!